eclair-core/src/main/scala/fr/acinq/eclair/wire/CommandCodecs.scala

```diff
@@ -27,18 +28,21 @@ object CommandCodecs {
   val cmdFulfillCodec: Codec[CMD_FULFILL_HTLC] =
     (("id" | int64) ::
       ("r" | bytes32) ::
-      ("commit" | provide(false))).as[CMD_FULFILL_HTLC]
+      ("commit" | provide(false)) ::
+      ("replytTo_opt" | provide(Option.empty[ActorRef]))).as[CMD_FULFILL_HTLC]
```

Comment:
nit (and same for the 2 others below):
```suggestion
      ("replyTo_opt" | provide(Option.empty[ActorRef]))).as[CMD_FULFILL_HTLC]
```

Reply:
Done with b80e0b50e.